src/js/models/layerstructure.js

```diff
@@ -609,6 +609,20 @@ define(function (require, exports, module) {
     LayerStructure.prototype.byID = function (id) {
         return this.layers.get(id, null);
     };
+    
+    /**
+     * Get a list of Layer models by layer IDs.
+     *
+     * @param {Immutable.Iterable.<number>} ids
+     * @return {Immutable.Iterable.<Layer>}
+     */
+    LayerStructure.prototype.byIDs = function (ids) {
+        return ids.reduce(function (layers, id) {
```

Comment:
I agree with that subtle behavior on removing null layers. I have changed the function to include null if any of the IDs is not found. Thanks @volfied for the code snippet, it is very useful!